content/docs/1.2.0/getting-started/quick-start.md

```diff
@@ -25,14 +25,14 @@ brew install iofogctl
 The Debian package can be installed like so:
 
 ```bash
-https://packagecloud.io/install/repositories/iofog/iofogctl/script.deb.sh | sudo bash
+curl https://packagecloud.io/install/repositories/iofog/iofogctl/script.deb.sh | sudo bash
```

Comment:
On the other iofogctl page we are using `curl -s http://xxxx` - we should make sure we are consistent.